server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java

```diff
@@ -476,9 +474,7 @@ private List<MeasurementPath> executeInternalCreateTimeseriesStatement(
     for (TSStatus subStatus : executionResult.status.subStatus) {
       if (subStatus.code == TSStatusCode.MEASUREMENT_ALREADY_EXIST.getStatusCode()) {
         alreadyExistingMeasurements.add(
-            (MeasurementPath)
-                PathDeserializeUtil.deserialize(
-                    ByteBuffer.wrap(subStatus.getMessage().getBytes())));
+            MeasurementPath.parseDataFromString(subStatus.getMessage()));
```

Comment:
subStatus.getMessage()  is measurementPathData ?

Reply:
Yes. When trying auto creating schema, the timeseries may be concurrently created by other threads or requests. The subStatus takes the existing schema info back without further schema fetch query.